cg/smart_contracts/ClearGDPR.sol

```diff
@@ -139,15 +156,10 @@ contract ClearGDPR {
 *       3. Only controllers can initiate data erasure events.
 *       4. The controller always has consent to use the user data. Else how could the user data be stored in the first place?
 *
-*   NOTES:
-*       1. We should move any processing of data to the back-end if possible. The only processing done in the smart contract is to ascertain that the data stored is valid.
-*       2. Controllers and processors have a different set of possible actions in the system. Hence the events each can fire are also different.
-*      We need a way to enforce this difference in the back-end if we stick with this smart contract.
-*       3. There's a bug when we use bytes32 instead of address to identificate the processors. The memory seems to not be cleared out. 
-*       
 *   TODO: 
 *       1. Change accessibility of setters to private (they are public for testing purposes)
 *       2. Implement the add/remove function of processor nodes in the network, or use a system ID for each processor, which would be simpler
 *       3. Study more about memory allocation/deallocation in Solidity. Will need to dive in inline Assembly
 *       4. Consider that each processor can delete their data of a subject, without the data being deleted from the whole network
+*       5. When a subjects gives consent, does that functions as a rectification of data? If not, his data should not be altered when he gives consent again
```

Comment:
good point!

Reply:
Probably anther issue to look in the future. Will create those